nats.go

```diff
@@ -2507,6 +2507,19 @@ func (nc *Conn) IsConnected() bool {
 	return nc.isConnected()
 }
 
+// Servers returns the list of known servers, including additional
```

Comment:
Hold on - let me test to see if the local user provided credentials are returned.